.travis.yml

```diff
@@ -12,11 +12,14 @@ before_install:
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > cc-test-reporter
   - chmod +x cc-test-reporter
   - ./cc-test-reporter before-build
+install:
+  - gem install rubocop
 before_script:
   - cp config/database.yml.travis config/database.yml
   - psql -c 'create database dontfile_test;' -U postgres
 script:
   - bundle install
+  - bundle exec rubocop
```

Comment:
Can you move this to after `bundle exec rails test` ? It's better run the stylesheet after the tests :smile: